config.go

```diff
@@ -29,42 +29,62 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
+const deviceConfigPath = "/etc/cacophony/device.yml"
```

Comment:
Can you please use the ".yaml" extension. That's what our other config files use. Both .yml and .yaml are correct for YAML files but let's keep things consistent.